index.html

```diff
@@ -219,6 +219,91 @@ <h3>Skills</h3>
 				</div>
 			</div>
 		</section>
+		<section id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
```

Comment:
- [x] Hi @zeff96, well done working on the contact section of your project, but according to the project requirement, we were asked to create a form-spree in such a way that any visitor can easily contact us when a message is submitted using the form-spree. but currently, the form-spree link is not working, kindly create an account on form-spree and use the id-link given to you in the `form action`

- I noticed nothing  happened when I click on the get in touch button to submit a message
- kindly fix it before resubmitting for review